### ISSUE TITLE
Portals don't start with win 10 and at this time none of my created portals i can make available/visible

### ISSUE THREAD

**Issue by Herwin2907** [Ross-Patterson/Portals-Desktop-Organization#92]
**Describe the bug**

**To Reproduce**
1. Start Windows (10)
2. Click on portals
3. Nothing happens first
4. I try to start Portals by clicking the Portals Icon and I get the general Portals screen, where I see all my defined potals which are still not visible.
5. I checked al the windows in the Visible box, but it has no effect

**Expected behavior**
Show my portals on the desktop

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows 10 Pro
 - Version 22H2

**Comment by origintopleft** [comment]
I'm having similar (but not identical issues), where in general, Portals seems to explode if the Windows desktop isn't available to draw on. The symptoms in my case are slightly different, possibly because I'm on Windows 11, but in general, it goes like this:

1. Log into Windows
2. Windows begins launching my startup programs and various user session processes (such as explorer.exe) in parallel.
3. Sometimes (I assume, I'm still getting the spinning "Welcome" message from winlogon at this point), the desktop loads first, and then Portals, and my portals function as intended. Sometimes, (again, assuming), Portals loads first, has no desktop to draw on, and then closes, requiring me to double click the Portals icon on my desktop. This is where my issue starts to diverge, because usually when I double click the icon it "just works".
4. I also had an update to my Nextcloud client this morning, which restarted explorer.exe, which caused Portals to disappear.  

I'm posting this here instead of starting a new issue because while my symptoms are slightly different, most of the malfunction is similar enough to yours that I think it might still be the same underlying issue. If it makes a difference to the developer, I'm also running Wallpaper Engine.

**Comment by wakitcha** [comment]
I'm seeing a similar issue, but nothing ever shows if you launch Portals multiple times.  Oddly there are multiple running processes in Task Manager.  It seems like we're both getting stuck during launch, but mine may be at a different location.  Have you had luck resolving this?

**Comment by Ross-Patterson** [comment]
Hi All,
Try upgrading to v3.0.0 (you might lose your settings unfortunately) - but It should work better.
